package.json

```diff
@@ -210,6 +210,7 @@
     "fuse.js": "^3.2.0",
```

Comment:
can we lose this dep or not yet?

Reply:
Yep, good point. It's not getting pulled into production anymore because of tree shaking (https://developer.mozilla.org/en-US/docs/Glossary/Tree_shaking). However, for clarity I should remove it from `package.json` too.